src/training/ea/lipizzaner_trainer.py

```diff
@@ -209,40 +217,77 @@ def _evaluate(self, generators: Population, discriminators: Population):
         fitness_labels = torch.squeeze(fitness_labels)
 
         # Evaluate generators
-        evaluate_fitness(
+        self.evaluate_fitness(
             attackers=generators,
             defenders=discriminators,
-            input_var=fitness_samples,
+            input_data=fitness_samples,
             fitness_mode=self.fitness_mode,
-            alpha=self.alpha,
-            beta=self.beta,
-            diverse_fitness=self.diverse_fitness,
         )
 
         # Evaluate discriminators
-        evaluate_fitness(
+        self.evaluate_fitness(
             attackers=discriminators,
             defenders=generators,
-            input_var=fitness_samples,
+            input_data=fitness_samples,
             fitness_mode=self.fitness_mode,
-            alpha=self.alpha,
-            beta=self.beta,
             labels=fitness_labels,
         )
 
+    def evaluate_fitness(
```

Comment:
Is it really necessary for this to go inside the class? It could receive `alpha`, `beta` and `diverse_fitness` and live in a helpers file, since it was already implemented to be standalone.

The less things inside the class, the better.

Reply:
If we have to leave it here, the name should start with `_` and differentiate this function clearly from `_evaluate`